src/Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI/Service/Impl/CustomApiMockCreator.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Extension;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Model;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Service.Interface;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.CLI.Service.Impl
```

Comment:
Can we change the folder structure from `Service.Impl` to `Services` and also change the namespace accordingly?

Reply:
Changed to match request in new commit